Dockerfile

```diff
@@ -1,29 +1,23 @@
-# Build Stage
-FROM node:22-bookworm-slim AS build
-
-WORKDIR /home/node/app
-
-COPY package*.json ./
-RUN npm install
-COPY . .
-RUN npm run build
-
-# Prod stage
 FROM node:22-bookworm-slim
 
 WORKDIR /home/node/app
 ENV NODE_ENV=production
 
 # Copy only needed files
-COPY  --chown=node:node --from=build /home/node/app/dist ./dist
-COPY  --chown=node:node --from=build /home/node/app/public ./public
-COPY  --chown=node:node --from=build /home/node/app/package*.json ./
+COPY  --chown=node:node ./package*.json .
 
 RUN npm ci --only=production --ignore-scripts
 
-COPY process.json .
+COPY --chown=node:node process.json .
 RUN mkdir -p /home/node/.pm2 /home/node/app/logs /home/node/app/pids && chown -R node:node /home/node/.pm2 /home/node/app/logs
 
+# db-migrate stuff
+COPY --chown=node:node migrations ./migrations
+COPY --chown=node:node database.json .
+
+COPY --chown=node:node ./public ./public
+COPY --chown=node:node ./src ./src
+
 USER node
```

Comment:
I'm not sure, but if you put `USER node` at the top of the file, you might get rid of all the `--chown=node:node`?

Reply:
Yes I copied this over from the original dockedfw here, but would indeed be interesting to check if this is actually needed.